apps/src/javalab/CommitDialogFile.jsx

```diff
@@ -0,0 +1,63 @@
+import React from 'react';
+import PropTypes from 'prop-types';
+import i18n from '@cdo/javalab/locale';
+import color from '@cdo/apps/util/color';
+import {fileShape} from './CommitDialogBody';
+
+const PADDING = 8;
+
+export default function CommitDialogFile({file, onToggleFile}) {
```

Comment:
nit: now that this is in its own file, `CommitDialogFile` is a confusing name. Could we change to `CommitDialogFileSelector` or something?

Reply:
Made it `CommitDialogFileRow`, how does that sound?